app/javascript/components/recipes/edit/recipe-edit.vue

```diff
@@ -43,6 +32,7 @@
       >
         1. {{ $t('msg.recipes.basic') }}
       </div>
+<<<<<<< HEAD
```

Comment:
⚠️ **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 6 spaces but found 0 spaces    vue/html-indent